pmd-swift/src/main/java/net/sourceforge/pmd/lang/swift/rule/bestpractices/ProhibitedInterfaceBuilderRule.java

```diff
@@ -0,0 +1,46 @@
+/**
+ * BSD-style license; for more info see http://pmd.sourceforge.net/license.html
+ */
+
+package net.sourceforge.pmd.lang.swift.rule.bestpractices;
+
+import java.util.List;
+
+import net.sourceforge.pmd.lang.ast.Node;
+import net.sourceforge.pmd.lang.swift.AbstractSwiftRule;
+import net.sourceforge.pmd.lang.swift.antlr4.SwiftParser;
+
+public class ProhibitedInterfaceBuilderRule extends AbstractSwiftRule<Boolean> {
```

Comment:
this rule looks like a prime case for using the rulechain (and showing it works with Antlr grammars!)